core/primitives/jwt/jwt.hpp

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <jwt-cpp/jwt.h>
+
+namespace fc::primitives::jwt {
+  using Permission = std::string;
```

Comment:
common_api.hpp only needs `Permission` type (maybe move it there), it doesnt need constants

Reply:
It would be used in API utils in next pr

Reply:
Is it to write permission list directly in method declaration?

Reply:
Yes